CDEFramework/Libraries/LegacyDictionarySupport/GlossaryAPIClient.cs

```diff
@@ -0,0 +1,152 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Net.Http;
+using System.Net.Http.Formatting;
+using Common.Logging;
+
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+using LegacyDictionarySupport.Configuration;
+
+namespace LegacyDictionarySupport
+{
+    public class GlossaryAPIClient
+    {
+        static ILog log = LogManager.GetLogger(typeof(GlossaryAPIClient));
+
+        private HttpClient _client = null;
+
+        /// <summary>
+        /// Creates a new instance of a Glossary API client.
+        /// </summary>
+        /// <param name="client">An HttpClient that has the BaseAddress set to the API address.</param>
+        public GlossaryAPIClient()
+        {
+            string baseApiPath = WebAPISection.GetAPIUrl();
+            string appPath = ConfigurationManager.AppSettings["GlossaryAPIAppPath"];
+            string versionPath = ConfigurationManager.AppSettings["GlossaryAPIVersionPath"];
+
+            if (string.IsNullOrWhiteSpace(appPath))
+                throw new ConfigurationErrorsException("error: GlossaryAPIAppPath cannot be null or empty");
+
+            if (string.IsNullOrWhiteSpace(versionPath))
+                throw new ConfigurationErrorsException("error: GlossaryAPIVersionPath cannot be null or empty");
+
+            HttpClient client = new HttpClient();
+            //NOTE: the base URL MUST have a trailing slash
+            client.BaseAddress = new Uri(String.Format("{0}/{1}/{2}/", baseApiPath, appPath, versionPath));
+
+            this._client = client;
+        }
+
+        /// <summary>
+        /// Calls the GetById endpoint of the Glossary API
+        /// </summary>
+        /// <param name="dictionary">Dictionary name (required)</param>
+        /// <param name="audience">Audience type(required)</param>
+        /// <param name="language">Language to use (required)</param>
+        /// <param name="id">The ID of the term</param>
+        /// <param name="useFallback">Whether or not to use the fallback logic</param>
+        /// <returns>A Best Bet result</returns>
```

Comment:
It doesn't return a Best Bet. 😉    There should also be a note that it returns `null` if no term is found.